package.json

```diff
@@ -13,15 +13,20 @@
     "@emotion/react": "^11.14.0",
     "@emotion/styled": "^11.14.0",
     "@fontsource/roboto": "^5.1.0",
+    "@fortawesome/fontawesome-free": "^6.7.2",
     "@mui/icons-material": "^6.2.0",
     "@mui/material": "^6.2.0",
+    "@mui/styled-engine-sc": "^6.2.0",
+    "@mui/x-charts": "^7.23.2",
     "axios": "^1.7.9",
     "jwt-decode": "^4.0.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-hook-form": "^7.54.0",
+    "react-hook-form": "^7.54.1",
+    "react-loading-indicators": "^1.0.0",
     "react-router-dom": "^7.0.2",
-    "react-toastify": "^10.0.6"
+    "react-toastify": "^10.0.6",
+    "styled-components": "^6.1.13"
```

Comment:
We need either styled components or emotion package but not both. In the documentation, MUI prefers emotion as it has support for server components. We can remove this package.